Dockerfile.openvino

```diff
@@ -124,7 +124,7 @@ tar xvf boost_1_69_0.tar.gz && cd boost_1_69_0 && ./bootstrap.sh && \
 --with-log --with-locale \
 install
 
-ARG OVMS_BRANCH="mediapipe_integration"
+ARG OVMS_BRANCH="storage_lib_fix"
```

Comment:
can we switch to main?

Reply:
Switching to main will introduce vulnerability for failures. I think we should switch to a release branch - after release, or use critical fixes branches - like in this example.

Reply:
We should declare specific sha of a commit from main here.

Reply:
Done.

Reply:
I would assume that we should keep the same sha in WORKSPACE & Dockerfile - add comment in both places about that. +1 after adding comment.